src/sourmash/distance_utils.py

```diff
@@ -0,0 +1,315 @@
+"""
+Utilities for jaccard/containment --> distance estimation
+Equations from: https://github.com/KoslickiLab/mutation-rate-ci-calculator
+Reference: https://doi.org/10.1101/2022.01.11.475870
+"""
+from dataclasses import dataclass, field
+from scipy.optimize import brentq
+from scipy.stats import norm as scipy_norm
+from numpy import sqrt
+from math import log, exp
+
+from .logging import notify
+
+def check_distance(dist):
+    if not 0 <= dist <= 1:
+        raise ValueError(f"Error: distance value {dist :.4f} is not between 0 and 1!")
+    else:
+        return dist
+
+def check_prob_threshold(val, threshold=1e-3):
+    """
+    Check likelihood of no shared hashes based on chance alone (false neg).
+    If too many exceed threshold, recommend user lower their scaled value.
+    # !! when using this, keep count and recommend user lower scaled val
+    """
+    exceeds_threshold = False
+    if threshold is not None and val > threshold:
+        notify("WARNING: These sketches may have no hashes in common based on chance alone.")
```

Comment:
for later consideration - we may want to switch these to Python [warnings](https://docs.python.org/3/library/warnings.html) at some point; I have tried to limit the use of 'notify' to code that is only called from the command line, as opposed to library code.

but I haven't used warnings enough to know how to use them properly and it's easy enough to change if this becomes an issue later :)

Reply:
issue filed :)

Reply:
thanks! https://github.com/sourmash-bio/sourmash/issues/1954